src/components/visualizations/chart/HighChartsRenderer.tsx

```diff
@@ -78,17 +83,22 @@ export default class HighChartsRenderer extends React.PureComponent<
         documentObj: document,
     };
 
+    private highcharsRenderertRef: any /*HTMLDivElement*/;
```

Comment:
Because `createRef` does not return full constructed object for `HTMLDivElement`. I got error like `missing properties` in returned object.
What we can do more is to add type to `createRef()` like
`React.createRef<HTMLDivElement>()`